src/promptflow-devkit/promptflow/_sdk/_service/static/chat-window/index.html

```diff
@@ -11,7 +11,7 @@
         padding: 0;
       }
     </style>
-    <script type="module" crossorigin src="/v1.0/ui/chat/assets/index-TbUGIzdW.js"></script>
+    <script type="module" crossorigin src="/v1.0/ui/chat/assets/index-T-QEXlSH.js"></script>
```

Comment:
please add screenshot in PR description what has changed.

Reply:
Updated